driver/flambda_backend_flags.ml

```diff
@@ -17,6 +17,7 @@ let use_ocamlcfg = ref true             (* -[no-]ocamlcfg *)
 let dump_cfg = ref false                (* -dcfg *)
 let cfg_invariants = ref false          (* -dcfg-invariants *)
 let regalloc = ref ""                   (* -regalloc *)
+let regalloc_linscan_threshold = ref 100_000 (* -regalloc-linscan-threshold *)
```

Comment:
It'd be nice to have `Flambda_backend_flags.default_regalloc_linscan_threshold` and print it in the command line option description because the units are somewhat mysterious (similar to `default_heap_reduction_threshold` for example).

Reply:
done